colearn_keras/keras_mnist.py

```diff
@@ -117,11 +150,13 @@ def prepare_resnet_learner(data_loaders: Tuple[PrefetchDataset, PrefetchDataset,
     return learner
 
 
-@FactoryRegistry.register_model_architecture("KERAS_MNIST", ["KERAS_MNIST"])
+@FactoryRegistry.register_model_architecture("KERAS_MNIST", ["KERAS_MNIST", "KERAS_MNIST_WITH_DP"])
 def prepare_learner(data_loaders: Tuple[PrefetchDataset, PrefetchDataset, PrefetchDataset],
                     steps_per_epoch: int = 100,
                     vote_batches: int = 10,
                     learning_rate: float = 0.001,
+                    diff_priv_config: Optional[DiffPrivConfig] = None,
+                    num_microbatches: int = 4,
```

Comment:
we use `num_microbatches` only for diff_priv, thus I guess it is better to set it also optional if `diff_priv_config` is optional

Reply:
that would mean we have to set it in the ui. and by name it might be used somewhere else, or not?

Reply:
usually for integer variables I don't like using Optional